README.rst

```diff
@@ -43,15 +58,20 @@ Find the shortest path between two nodes in an undirected graph:
 Install
 -------
 
-Install the latest version of NetworkX::
+Install the latest released version of NetworkX:
+
+.. code:: shell
 
     $ pip install networkx
 
-Install with all optional dependencies::
+Install with all optional dependencies:
+
+.. code:: shell
 
     $ pip install networkx[all]
 
-For additional details, please see `INSTALL.rst`.
+For additional details,
+please see the `installation guide <https://networkx.org/documentation/latest/install.html>`_.
```

Comment:
For this and the contributor guide below, I would probably link to the `stable` docs instead of `latest`. What do y'all think?

Reply:
I don't feel strongly either way. I went with `latest` because I figured any changes between the last release and the current `main` _for the installation guide_ were likely to be backward-compatible (and so hopefully just a net benefit for the reader), but I could see how once a user completes the `latest` installation guide, they'd then go on to read the `latest` docs, which could be more problematic.

I believe for the contributor guide there is probably value in keeping `latest` because it makes sense to lead them to the version of the docs they'd be working on/with when contributing, but again, I'll happily go with whatever the consensus ends up being.